app/pulp/app/serializers/repository.py

```diff
@@ -46,11 +46,72 @@ class ImporterSerializer(MasterModelSerializer):
 
     Every subclass must override the `_href` field with a `RepositoryNestedIdentityField` that
     defines the view_name.
+
+    `repository` is defined on the `Importer` model, but is excluded here because the user must
+    already know about a `Repository` and its relationship to the Importer because the REST
+    endpoint includes `Repository.name`.
```

Comment:
The part
> user must already know about a `Repository`...

is not very clear for me.
What about something like "excluded because related repository is specified in the URL (or API endpoint )"? And it  implies that user just can't even make a request without knowing it, and at the same time this info is available to the viewsets to be used.
I do not feel strong about it though, maybe it is just me :)